modules/admin_manual/pages/maintenance/manual_upgrade.adoc

```diff
@@ -166,41 +134,46 @@ code) and the Market app is enabled, but there is no available internet
 connection, then these apps will need to be manually updated once the
 upgrade is finished.
 
+[[copy-old-apps]]
+== Copy Old Apps
+If you are using third party or enterprise applications, look in your new `/var/www/owncloud/apps/` directory to see if they are there.
```

Comment:
`directory to see if they are there` --> `directory to see if the apps are present`